src/routes/safe/components/Settings/SafeDetails/style.ts

```diff
@@ -23,4 +23,8 @@ export const styles = () => ({
   versionNumber: {
     height: '21px',
   },
+  link: {
+    color: `${fontColor}`,
+    borderBottom: `1px solid ${fontColor}`,
```

Comment:
This should be
`text-decoration: underline`

Reply:
fixed